.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,10 +1,13 @@
 #### Before the PR (what is the issue or what needed to be done)
+
 #### The solution
+
 #### Screenshots (if appropriate)
 
-## Checklist
-* [ ] Connect the issue
-* [ ] Test plan
-* [ ] Copyright header
-* [ ] Code formatting
-* [ ] Documentation
+#### Reviewer Checklist
+* [ ] Added an explanation about the work done
```

Comment:
This is already part of the template no? i.e. the section _The solution_

Reply:
it is but this is a reminder for the reviewer and also for the person who creates it to make sure they filled out the information